CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+# 0.44.1 / 2020-05-31
```

Comment:
`2021`, may be true for previous releases as well

Reply:
good catch! Addressed on all 2021 releases 👍